examples/outer-join-clause/outer_join_clause.out

```diff
@@ -0,0 +1,2 @@
+$ bal run outer_join_clause.bal
```

Comment:
```suggestion
$ bal run outer_join_clause.bal
```
The command should be added in the `.md` file and only the output should be there in the `.out` file.